closed/GensrcJ9JCL.gmk

```diff
@@ -78,24 +86,12 @@ $(J9JCL_SOURCES_DONEFILE) : $(AllJclSource) $(AllDdrSource) $(OPENJDK_STAGED_PRO
 		JAVA_HOME=$(BOOT_JDK) \
 		preprocessor
 	@$(ECHO) Generating J9JCL sources
-	$(call jpp_generate_sources,$(call FixPath,$(OPENJ9_TOPDIR)),jcl)
-	$(call jpp_generate_sources,$(call FixPath,$(SUPPORT_OUTPUTDIR)),overlay)
+	$(call jpp_generate_sources,,$(SUPPORT_OUTPUTDIR),overlay,JAVA$(VERSION_FEATURE),$(J9JCL_SOURCES_DIR))
+	$(call jpp_generate_sources,$(JPP_INCLUDE_ALL_JAVA),$(TOPDIR),closed,JAVA$(VERSION_FEATURE),$(J9JCL_SOURCES_DIR))
+	$(call jpp_generate_sources,,$(OPENJ9_TOPDIR),jcl,JAVA$(VERSION_FEATURE),$(J9JCL_SOURCES_DIR))
   ifeq (true,$(OPENJ9_ENABLE_DDR))
 	@$(ECHO) Generating DDR_VM sources
-	@$(BOOT_JDK)/bin/java \
-		-cp "$(call FixPath,$(JPP_JAR))" \
-		-Dfile.encoding=US-ASCII \
-		com.ibm.jpp.commandline.CommandlineBuilder \
-			-includeIfUnsure \
-			-noWarnIncludeIf \
-			-verdict \
-			-baseDir "$(call FixPath,$(OPENJ9_TOPDIR)/debugtools)/" \
-			-config DDR_VM \
-			-srcRoot DDR_VM/ \
-			-xml "$(call FixPath,$(OPENJ9_TOPDIR)/jcl/jpp_configuration.xml)/" \
-			-dest "$(call FixPath,$(J9JCL_SOURCES_DIR)/openj9.dtfj/share/classes)" \
-			-macro:define "JAVA_SPEC_VERSION=$(VERSION_FEATURE)" \
```

Comment:
The option `-macro:define "JAVA_SPEC_VERSION=$(VERSION_FEATURE)"` should be retained for DDR_VM.

Reply:
Added the option into `jpp_generate_sources` which is not harmful to other configurations already having such macro define.

Reply:
Thanks @keithc-ca suggestions, this is ready for another look.